File: 3019-furthest-point-from-origin/furthest-point-from-origin.py
class Solution:
    def furthestDistanceFromOrigin(self, moves: str) -> int:
        balance = 0
        blank = 0

        for ch in moves:
            if ch == 'L':
                balance -= 1
            elif ch == 'R':
                balance += 1
            else:
                blank += 1

        return abs(balance) + blank

File: 3019-furthest-point-from-origin/furthest-point-from-origin.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
         int balance = 0;
        int blanks = 0;

        for(char c : moves.toCharArray()){
            if(c == 'L') balance--;
            else if(c == 'R') balance++;
            else blanks++;
        }

        return Math.abs(balance) + blanks;
    }
}